packages/easysearch:core/lib/core/search-collection.js

```diff
@@ -184,20 +184,40 @@ class SearchCollection {
       this.added(collectionName, 'searchCount' + definitionString, { count });
 
       let intervalID;
-
       if (collectionScope._indexConfiguration.countUpdateIntervalMs) {
         intervalID = Meteor.setInterval(
           () => this.changed(
             collectionName,
             'searchCount' + definitionString,
-            { count: cursor.mongoCursor.count() }
+            // TODO: I substituted cursor.mongoCursor.count with cursor.count. will this break things?
+            { count: cursor.count && cursor.count() || 0 }
```

Comment:
I think you'd have to check out all useages of the core `Cursor` and check if it's simply the `mongoCursor.count()` that's passed in. I'm not sure about that